### PR TITLE
Feature/lib 43

### PR DESCRIPTION
**Pull request by Emilo067**
# Pull Request - Implement Alert Component

## Description

This pull request introduces the `Alert` component, designed to display success and error messages with a close button. The component is flexible and can handle different types of alerts, such as success and error notifications, by passing the appropriate props.

### Main Changes

- **Alert Component**:
  - Added `Alert` component that accepts `type`, `message`, and `onClose` props.
  - Supports `success` and `error` alert types with corresponding styles.
  - Includes a close button that triggers the `onClose` callback.
  - Utilizes `Card`, `Button`, and `Typography` components for consistent styling.
- **Storybook**:
  - Added Storybook stories to demonstrate the usage of the `Alert` component with different types of alerts.
  - Created a controlled story to showcase the alert being dynamically shown and hidden.
- **Styles**:
  - Added SCSS module for styling the `Alert` component based on the alert type.
  
### How to Test

1. Pull the branch and run the project.
2. Open Storybook by running `npm run storybook`.
3. Navigate to the `Alert` component in Storybook.
4. Test the component by clicking the "Show" button to display the alert and then close it using the "X" button.

#### Success Alert
Success Alert Screenshot

#### Error Alert
Error Alert Screenshot

### Additional Notes

- Ensure that the `Alert` component aligns with the overall design system.
- Tested responsiveness and ensured compatibility with various screen sizes.

### Linked Issues

- Closes #123 (Replace with the actual issue number if applicable)

### Checklist

- [x] Component is tested and works as expected.
- [x] Code has been linted and follows coding standards.
- [x] Documentation is updated or added where necessary.
- [x] Storybook stories have been added or updated.

---

Please review the code and let me know if any changes are required. Thank you!

### CODE REVIEW

File: src/components/ui/alert/alert.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * @example
+ * // Example usage of the Alert component
+ * <Alert
+ *   type="success"
+ *   message="Your settings have been saved!"
+ *   onClose={() => console.log('Alert closed')}
+ * />
+ */
+
+export const Alert: FC<AlertProps> = ({ type, message, onClose }) => {
```
Comment (Lelik7777) [inline src/components/ui/alert/alert.tsx]: лучше не использовать FC component

File: src/components/ui/alert/alert.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const classNames = {
```
Comment (Lelik7777) [inline src/components/ui/alert/alert.tsx:31]: add className in props

Comment (Emilo067) [reply src/components/ui/alert/alert.tsx:31]: исправил

File: src/components/ui/alert/alert.tsx (hunk 1 above)
Comment (Emilo067) [reply src/components/ui/alert/alert.tsx]: исправил

**Review by Lelik7777 — APPROVED** [review]

**Review by NoName-ForTeam — APPROVED** [review]

**Review by EvgeniyaYakunina — APPROVED** [review]

**Pull request merged**